### ISSUE TITLE
Allow personal answers notification disappears after tapping in Android

### ISSUE THREAD

**Issue by mboos** [tronikos/google_assistant_sdk_custom#8]
When I tap the "Allow personal answers" notification that appears on Android, the screen sort of flashes as if an app is being opened and closed, and then I'm back at whatever app was opened before. There is no place to enter a device name. I'm not seeing anything new in my Google security activity on my phone.

**Comment by tronikos** [comment]
This happens when the credentials are not created with application type of desktop app. Follow the steps again.

**Comment by mboos** [comment]
The credentials are desktop application type though! I did try creating a second set of credentials in the console, with the exact same result.

**Comment by tronikos** [comment]
Maybe the credentials file wasn't placed under /config/google_assistant_sdk_credentials.json ?

Try running demo.py of the underlying library at https://github.com/tronikos/gassist_text to ensure you are using the proper credentials.

**Comment by PBogi** [comment]
Had the same issue on my phone as well as tablet and also with gassist_text.
I found a workaround [here](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues/863#issuecomment-1474980944) (relevant step 3-1):

- Use a phone or emulator (e.g MEmu**) with Android 7.1.1
- Sign in with your Google Account
- Install/Update Google and Google Assistant App

With this Android Version I was able to open the notification and set a name.

Hope this helps!

** MEmu does not look very trustworthy to me and is filled with ads. Be careful to deselect any adware when installing and I also revoked any connections to my google account from this device after I was done.
